src/nvim/os/win_defs.h

```diff
@@ -3,6 +3,8 @@
 
 #include <windows.h>
 
+#include <uv.h>
```

Comment:
Ok. If we are fine with anything in `os/*` including `uv.h` then we can drop the `uv.h` include.

I believe we will need to add a `typedef` for `ssize_t` then though so that everything outside of `os/*` can see the type  right?